src/Compilers/CSharp/Portable/Binder/ForEachLoopBinder.cs

```diff
@@ -612,7 +612,8 @@ private BoundExpression UnwrapCollectionExpressionIfNullable(BoundExpression col
                     return BoundCall.Synthesized(
                         syntax: exprSyntax,
                         receiverOpt: collectionExpr,
-                        method: nullableValueGetter);
+                        method: nullableValueGetter,
+                        binder: this);
```

Comment:
>binder: this); [](start = 24, length = 14)

I do not think this change belongs in this PR

Reply:
Separated out into [https://github.com/dotnet/roslyn/pull/49885](https://github.com/dotnet/roslyn/pull/49885).

---
In reply to: [539666484](https://github.com/dotnet/roslyn/pull/49804#discussion_r539666484) [](ancestors = 539666484)